examples/model-viewer/shaders.slang

```diff
@@ -458,14 +447,14 @@ float4 fragmentMain(
     // a BRDF suitable for evaluating under illumination
     // from different light sources.
     //
-    // Note that the return type here is `TMaterial.BRDF`,
+    // Note that the return type here is `gMaterial.BRDF`,
     // which is the `BRDF` type *associated* with the (unknown)
     // `TMaterial` type. When `TMaterial` gets substituted for
     // a concrete type later (e.g., `SimpleMaterial`) this
     // will resolve to a concrete type too (e.g., `SimpleMaterial.BRDF`
     // which is an alias for `BlinnPhong`).
     //
-    TMaterial.BRDF brdf = gMaterial.prepare(g);
+    let brdf = gMaterial.prepare(g);
```

Comment:
Out of curiosity, does it work to write `gMaterial.BRDF` here? I would hope that it Just Works in Slang to use types that are members of values, but I haven't checked how well the compiler handles it.